app.js

```diff
@@ -1,2 +1,30 @@
 const Discord = require('discord.js');
-require('dotenv').config();
\ No newline at end of file
+require('dotenv').config();
+
+const client = new Discord.Client();
+client.on('ready', () => {
+    console.log('The bot is connected to Yang\'s Gang!');
+});
+client.login(process.env.BOT_TOKEN)
+
+client.on('message', (msg) => {
+    if (msg.content === 'Hello ICDB!') msg.reply('Hi :)');
+
+    if(msg.content == '!date') {
+        let date = new Date();
+        let content = 'Today\'s date is ' + date.getMonth() + '/' + date.getDate() + '/' + date.getFullYear() + '.';
+        msg.channel.send(content)
+    }
+
+    if(msg.content == '!time') {
```

Comment:
replace all the let and var with `const` except for `var amOrPm`. Since that one can actually change (in the if statement), it should be `let amOrPm`

Reply:
Keeping  let hours as let hours instead of const hours since hours is modified depending on am or pm.

Reply:
perfect